File: samples/client/petstore/java/retrofit2-play26/src/test/java/org/openapitools/client/api/AnotherFakeApiTest.java
package org.openapitools.client.api;

import org.openapitools.client.ApiClient;
import org.openapitools.client.model.Client;
import java.util.UUID;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Disabled;
import org.junit.jupiter.api.Test;

import java.time.LocalDate;
import java.time.OffsetDateTime;
import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;

import jakarta.validation.constraints.*;
import jakarta.validation.Valid;

/**
 * API tests for AnotherFakeApi
 */
public class AnotherFakeApiTest {

    private AnotherFakeApi api;

    @BeforeEach
    public void setup() {
        api = new ApiClient().createService(AnotherFakeApi.class);
    }

    /**
     * To test special tags
     *
     * To test special tags and operation ID starting with number
     */
    @Test
    public void call123testSpecialTagsTest() {
        UUID uuidTest = null;
        Client body = null;
        // Client response = api.call123testSpecialTags(uuidTest, body);

        // TODO: test validations
    }
}

File: samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/test/another_fake_api_test.dart
import 'package:openapi/api.dart';
import 'package:openapi/api/another_fake_api.dart';
import 'package:test/test.dart';


/// tests for AnotherFakeApi
void main() {
  final instance = Openapi().getAnotherFakeApi();

  group(AnotherFakeApi, () {
    // To test special tags
    //
    // To test special tags and operation ID starting with number
    //
    //Future<Client> 123test@$%SpecialTags(Client client) async
    test('test 123test@$%SpecialTags', () async {
      // TODO
    });

  });
}
